docs/index.md

```diff
@@ -0,0 +1,13 @@
+# Base9 Documentation 
+
+## Introduction to Base9
+
+Base9 is a tiny virtual machine and programming language! We’re using it to show people how to use [Eclipse OMR] to build their own programming language. 
+
+[Eclipse OMR]: https://github.com/eclipse/omr
+
+The Base9 language is a subset of JavaScript and is very limited in functionality. It currently only contains support for integers. 
+
+For a full overview of Base9 and it's components, plus a tutorial on adding new bytecodes, see our [Supercharge a Language Runtime] presentation and tutorial. 
+
+[Supercharge a Language Runtime]: ./presentations/CASCON2017_SuperchargeALanguageRuntime.pdf
```

Comment:
I think we should link to the Developer Journey and the Setup Page from here. This is going to be the websites main landing page right? @rwy0717

Reply:
100% agree. I haven't written any new content, just pulled in some of the setup & intro stuff to the index. That would be my very next PR, unless you want me to just do it now 👍